src/jsmind.js

```diff
@@ -417,7 +420,7 @@ export default class jsMind {
      * @param {string} node_id
      * @param {string} topic
      * @param {Record<string, any>=} data
-     * @param {number=} direction
+     * @param {'left'|'center'|'right'=} direction
```

Comment:
'-1', '0', '1', and number are also allowed. see
https://github.com/hizzgdev/jsmind/blob/master/src/jsmind.common.js#L28